channels-and-providers/default-providers.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Novu Providers'
+description: 'Learn about the default Novu email and sms providers'
+icon: 'code'
+---
+
+
+## Novu Providers
+
+To help you evaluate our services better, Novu provides an email and sms provider by default for every account. After signing up, you can go to the [Integrations store](https://web.novu.co/integrations) on the Novu web dashboard to see this. 
+
+
+## Novu Email Provider
+If you've a newly signed-up account on Novu, it will look like this:
+
+![Email providers on Novu web dashboard's Integration store](https://res.cloudinary.com/dxc6bnman/image/upload/v1691175129/guides/SCR-20230805-bhlg_k6ents.png)
```

Comment:
The integration store is now different. Please take a new screenshot and update the image.